packages/dotnet/ReadMe/ConfigValues.cs

```diff
@@ -1,23 +1,31 @@
-﻿using ReadMe.HarJsonObjectModels;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
+using ReadMe.HarJsonObjectModels;
```

Comment:
StyleCop prefers `System` imports to come before anything else, and also be alphabetized. https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1208.md